### PR TITLE
Lift legacy check for Access List and Access Monitoring

### PR DESCRIPTION
**Pull request by kimlisa**
part of https://github.com/gravitational/teleport.e/issues/2317

Legacy (`non-usage based`) on-prem and cloud now gets AL and AM limited as default starting `v15` (previously we allowed unlimited access)

### CODE REVIEW

File: api/client/webclient/webconfig.go [hunk 1]
```diff
@@ -97,6 +97,9 @@ type FeatureLimits struct {
 	// Defines the max number of days to include in an access report if
 	// feature is not enabled.
 	AccessMonitoringMaxReportRangeLimit int `json:"accessMonitoringMaxReportRangeLimit"`
+	// MonthlyRequestLimit is the usage-based limit for the number of
+	// access requests created in a calendar month.
+	AccessRequstMonthlyRequestLimit int `json:"accessRequstMonthlyRequestLimit"`
```
Comment (justinas) [inline api/client/webclient/webconfig.go:102]: Requst -> Request.

File: web/packages/teleport/src/config.ts [hunk 2]
```diff
@@ -71,6 +71,7 @@ const cfg = {
   featureLimits: {
     accessListCreateLimit: 0,
     accessMonitoringMaxReportRangeLimit: 0,
+    accessRequstMonthlyRequestLimit: 5,
```
Comment (flyinghermit) [inline web/packages/teleport/src/config.ts:74]: I thought his would come from web API?

Comment (kimlisa) [reply web/packages/teleport/src/config.ts:74]: yes you're right, oversight, made it zero value

**Comment by kimlisa** [comment]
friendly ping @justinas @flyinghermit

**Review by flyinghermit — APPROVED** [review]

**Review by justinas — APPROVED** [review]

**Review by r0mant — APPROVED** [review]

**Pull request merged**